migrate/mysql/000001_initial_schema.up.sql

```diff
@@ -16,6 +16,7 @@ CREATE TABLE message_message (
   id TEXT NOT NULL,
   message TEXT NOT NULL,
   description TEXT,
+  positions TEXT,
```

Comment:
In smaller projects, it may be sufficient to store an extracted text in a single TEXT field if it only exists in one or two files. However, in larger projects where the same translation is likely to appear in multiple files, creating a separate table to track their positions could be a more efficient solution? 

I am just brainstorming here, maybe I am wrong :)

Reply:
The probability is zero that the value gets even close to the limit. It's good.

Reply:
why not JSON?

Reply:
for writing SQL queries to target specific source files, maybe could be useful.

Reply:
I didn't really consider JSON as an option, was more on the fence about creating a new table.